react/NestedSelect/NestedSelect.jsx

```diff
@@ -186,7 +225,20 @@ NestedSelect.propTypes = {
    * const transformParentItem = item => ({ ...item, title: "Everything"})
    * ````
    */
-  transformParentItem: PropTypes.func
+  transformParentItem: PropTypes.func,
+
+  /**
+   * Search options defines :
+   * - placeholder in input search
+   * - the implementation of a search
+   * - how to display the results of a search
+   */
+  searchOptions: PropTypes.shape({
+    placeholderSearch: PropTypes.string.isRequired,
+    noDataLabel: PropTypes.string.isRequired,
+    onSearch: PropTypes.func.isRequired,
+    displaySearchResultItem: PropTypes.func.isRequired
```

Comment:
PropTypes.node

Reply:
Done

Reply:
in fact no it is a function that I use and not a component